Kalarba server without autofac

arjun-nakkeeran opened the pull request:
This PR adds a new sample project "KalarbaMSDI.Server" which is a port of Kalarba server without autofac dependency. Instead it uses Microsoft.Extensions.DependencyInjection

A little background why this PR: In our project we didn't use Autofac and found it difficult in the beginning to configure the Kalarba server without autofac. I managed to run it with just Microsoft dependency injection after all. Thought it would be helpful for someone like me to get started with Openiddict and Owin.

kevinchalet commented:
Thanks for your PR! Since the KalarbaMSDI.Server is a clone of the existing sample, would it instead make sense to add a code comment in `Program.cs` to mention that while Autofac is the recommended option, MS.Ext.DI can be used by following the instructions mentioned in https://kevinchalet.com/2020/03/03/adding-openiddict-3-0-to-an-owin-application/?

arjun-nakkeeran commented:
Yes, Since the KalarbaMSDI is a clone, it makes sense to add a code comment in original project. But I felt the instruction in blog post for "without autofac" was missing few configuration. 
For eg,
- In Webapi configuration, instead of AutofacWebApiDependencyResolver, we were supposed to have a generic resolver like I added here. [DefaultDependencyResolver.cs](https://github.com/openiddict/openiddict-samples/pull/145/files#diff-99656c3435754f4edab531bec2b3ad9093fe184ea63d23677d939b346d7cafb4)
- For validation, just added this line `app.UseOpenIddictValidation()` in `Startup.cs`

May I suggest to update the blogpost with these changes?

kevinchalet commented:
> May I suggest to update the blogpost with these changes?

Thanks for your suggestion: I updated the blog post to include hints on how to configure token validation with Web API and added links to the 2 OWIN/ASP.NET 4.x samples.
